helpers/date.test.js

```diff
@@ -0,0 +1,11 @@
+import { formatDate } from './date';
+
+describe('Date helper', () => {
+  describe('formatTodaysDate', () => {
```

Comment:
```suggestion
  describe('formatDate', () => {
```

Reply:
done in 664dcd34571dbbbd6d26d745024c495892f75226